mkdocs.yml

```diff
@@ -10,9 +10,14 @@ edit_uri: edit/trunk/docs/
 nav:
   - Getting Started:
       - index.md
-      - quickstart.md
-      - installing-kolena.md
-      - building-a-workflow.md
+      - Quickstart: quickstart.md
+      - Installing <code>kolena</code>: installing-kolena.md
+      - Building a Workflow: building-a-workflow.md
+      - Core Concepts:
+          - core-concepts/index.md
+          - Workflow: core-concepts/workflow.md
+          - Test Case & Test Suite: core-concepts/test-suite.md
+          - Model: core-concepts/model.md
```

Comment:
Flagging that it doesn't feel like `workflow`, `Test Case & Test Suite`, and `Model` are children of `Core Concepts` -- wonder if there's any way to indent them.
![Screenshot 2023-06-12 at 17 20 42](https://github.com/kolenaIO/kolena/assets/2481598/73d2ba8b-abb5-4312-9ec4-43e0a0d56e93)

Reply:
Addressing in a follow-up. The content isn't quite ready yet